api/src/main/java/org/openmrs/module/eptsreports/reporting/library/queries/BaseQueries.java

```diff
@@ -24,8 +24,7 @@
 	// they map to concept_id=1369 - TRANSFER FROM OTHER FACILITY
 	// TODO: Query needs to be refactored
 	public static String getBaseCohortQuery(Map<String, String> parameters) {
-		String query = "select patient_id from "
-		        + "(select p.patient_id from patient p join encounter e on e.patient_id=p.patient_id join person pr on pr.person_id = p.patient_id "
+		String query = "select p.patient_id from patient p join encounter e on e.patient_id=p.patient_id join person pr on pr.person_id = p.patient_id "
 		        + "where e.voided=0 and p.voided=0 and e.encounter_type in (%s) and e.encounter_datetime<=:endDate and e.location_id = :location and pr.birthdate is not null "
```

Comment:
@ynurmahomed this query was updated in this commit: 7e326c4773aab0837e252ac8128d5ce0398d62be

Please pull that change in.

Reply:
@nthfloor done